File: examples-sdk/python/603-db-clickhouse-batch.py
#!/usr/bin/env python3
"""603-db-clickhouse-batch — Python SDK equivalent
Compile: vil compile --from python --input 603-db-clickhouse-batch.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()

File: examples-sdk/typescript/603-db-clickhouse-batch.ts
#!/usr/bin/env tsx
// 603-db-clickhouse-batch — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 603-db-clickhouse-batch.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();
